### PR TITLE
Make Shapes responsive for different screen width

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR follows up #11508 

Screen Shot 2024-08-13 at 22 18 39

Slack Conversation: https://layer5io.slack.com/archives/C04LJS10VTL/p1723556660685669?thread_ts=1723555576.935239&cid=C04LJS10VTL

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d06d3bcb9243a618313ce135448ec1b56d174a9c at: https://66bb951ebe029a77edbee5c6--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 96a954a86e1cf759dfc4e3e3f708b3d7127b06a2 at: https://66bb961b989906788ee65136--meshery-docs-preview.netlify.app

File: docs/pages/extensions/meshmap/component-shape-guide.md [hunk 1]
```diff
@@ -21,69 +21,75 @@ Below are all the shapes with their current usage in a general context.
...
+      <div class="shapes-card">
+      <div class="shapes-svg-container">
+      {% if shape.SVG %}
+        <img src="{{ site.baseurl }}/assets/shapes/{{shape.SVG}}" alt="Shape" class="shapes-svg">
+      {% endif %}
+       <div style="text-align:center;">{{ shape.Shape }}</div>
+      </div>
+      <div class="shapes-details">
+      <div class="shapes-description">{{ shape.Description }}</div>
+      <div class="shape-usage">{{ shape.CommonUseCase }}</div>
```
Comment (vishalvivekm) [inline docs/pages/extensions/meshmap/component-shape-guide.md:90]:
```suggestion
      <div>{{ shape.CommonUseCase }}</div>
```

File: docs/pages/extensions/meshmap/component-shape-guide.md [hunk 2]
```diff
@@ -21,69 +21,75 @@ Below are all the shapes with their current usage in a general context.
...
+
+      <div class="shapes-card">
+      <div class="shapes-svg-container">
+      {% if shape.SVG %}
+        <img src="{{ site.baseurl }}/assets/shapes/{{shape.SVG}}" alt="Shape" class="shapes-svg">
+      {% endif %}
+       <div style="text-align:center;">{{ shape.Shape }}</div>
+      </div>
+      <div class="shapes-details">
+      <div class="shapes-description">{{ shape.Description }}</div>
```
Comment (vishalvivekm) [inline docs/pages/extensions/meshmap/component-shape-guide.md:89]:
```suggestion
      <div>{{ shape.Description }}</div>
```

File: docs/pages/extensions/meshmap/component-shape-guide.md [hunk 3]
```diff
@@ -21,69 +21,75 @@ Below are all the shapes with their current usage in a general context.
...
-      </tr>
+
+<div class="shapes-container">
+{% assign sorted_shapes = site.data.shapes | sort: 'Shape' %}
+    {% for shape in sorted_shapes %}
+
+      <div class="shapes-card">
+      <div class="shapes-svg-container">
+      {% if shape.SVG %}
+        <img src="{{ site.baseurl }}/assets/shapes/{{shape.SVG}}" alt="Shape" class="shapes-svg">
```
Comment (vishalvivekm) [inline docs/pages/extensions/meshmap/component-shape-guide.md:84]:
```suggestion
        <img src="{{ site.baseurl }}/assets/shapes/{{shape.SVG}}" alt="Shape">
```

**Comment by l5io** [comment]
🚀 Preview for commit 660df44a2774f06f3d36066974f8f3b35ec256f5 at: https://66bbd4b7e9da4eb71da5fe5c--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**